uiowa/uiowa
# Hcaptcha work

Resolves #7394 

# How to test

- For testing locally, DM Marie for necessary settings file changes.
- `ddev composer install && ddev blt ds --site=admissions.uiowa.edu && ddev drush @admissions.local uli /admin/structure/webform`
- You can create a new webform or click 'Build' on an existing one.
- Click Add Element, then find CAPTCHA under Advanced Elements and add it. 
- If current role is admin, check the Admin mode box to see the captcha as an admin and save. 
- View the webform, the hCaptcha should be appearing now. Trying submitting without filling it out, you should receive an error message. Then see if you can pass it. (Bonus question: Does the challenge level seem right or should it be adjusted?) 
- Confirm non-logged in users can see the hCaptcha on the page.
- Masquerade as a webmaster and confirm webmasters can also add hCaptchas to webforms. (Looks like webmaster is the only other role that can create/edit webforms at this time.)

# Next steps
- Enable this on the uiowa02 dev environment
- Send a link to TM for accessibility testing and ISPO for security testing
